client/mysqlbinlog.cc

```diff
@@ -734,8 +738,8 @@ static void convert_path_to_forward_slashes(char *fname)
 
 
 /**
-  Indicates whether the given database should be filtered out,
-  according to the --database=X option.
+  Indicates whether the given databases should be filtered out,
+  according to the --database=X or --databases=X1,X2,X3 option.
```

Comment:
I think this is a residual line from the previous version of the path. No need for `--databases` anymore right?